src/helm/benchmark/metrics/paraphrase_generation_metrics.py

```diff
@@ -0,0 +1,44 @@
+from typing import List
+
+from helm.benchmark.adaptation.request_state import RequestState
+from .metric import Metric
+from .metric_name import MetricName
+from .statistic import Stat
+from nltk.translate.bleu_score import corpus_bleu
+
+
+class CLEVAParaphraseGenerationMetric(Metric):
+    """
+    Compute the Chinese iBLEU score for Paraphrase Generation scenarios of CLEVA benchmark.
+    This implementation allows variable number of references (i.e., golds).
+    If there are more than one hypothesis (i.e., preds), only the first one is adopted in the calculation.
+
+    Reference:
+    https://aclanthology.org/2022.acl-long.178.pdf
+    https://aclanthology.org/P12-2008.pdf
+    """
+
+    def evaluate_instances(self, request_states: List[RequestState]) -> List[Stat]:
+
+        inputs: List = []
+        preds: List = []
+        golds: List[List[str]] = []
+
+        for request_state in request_states:
+            inputs.append(request_state.instance.input.text)
+
+            assert request_state.result is not None
+            preds.append(request_state.result.completions[0].text)
+
+            golds.append([reference.output.text for reference in request_state.instance.references])
+
+        tokenized_inputs = [[[i for i in input]] for input in inputs]
```

Comment:
Please add a comment here to mark that we are using characters for computing BLEU.